### PR TITLE
Fixed migration relavant issues and added some limitations to CI

### PR DESCRIPTION
**Pull request by CihatAltiparmak**
With this PR, The issues related to migration to moveit organization is solved.

### CODE REVIEW

**Review by sjahr — COMMENTED** [review]
Makes sense, what is necessary for the last CI Job to pass?

**Comment by CihatAltiparmak** [comment]
Benchmarks inside CI uses the docker image created by github actions. This dockerfile always clones from main branch. When we merge this PR, i hope this CI will work.

File: .github/workflows/create_docker_image.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 name: Create Docker Image
 
-on: [push, pull_request, workflow_dispatch]
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+  schedule:
+    - cron: '03 5 * * *'
```
Comment (CihatAltiparmak) [inline .github/workflows/create_docker_image.yml:8]: What's your opinion for this change, @sjahr ?

Comment (CihatAltiparmak) [reply .github/workflows/create_docker_image.yml:8]: I am about to forget this PR. Just for friendly pinging, @sjahr

**Review by sjahr — APPROVED** [review]
Sorry, as you stated in your comment I forgot this :see_no_evil: but I think it makes sense. Go ahead!

**Pull request merged**